src/Breweries.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Interns2022B;
+
+class Breweries
+{
+    public const SEARCH = "choice";
+
+    public function getBreweries(array $breweriesData): array
```

Comment:
instead operations on array, refactor it to use Collection of brewery object which are returned from brewery factory